apps/web-giddh/src/app/tb-pl-bs/components/pl/pl.component.ts

```diff
@@ -151,7 +151,7 @@ export class PlComponent implements OnInit, AfterViewInit, OnDestroy {
                         cg.childGroups = [];
                         if (['purchasesStockAmount', 'manufacturingExpenses'].includes(f)) {
                             cg.groupName = `+ ${cg.groupName}`;
-                        } else if (f === 'closingInventory') {
+                        } else if (['closingInventory', 'debitNoteStockAmount'].includes(f)) {
```

Comment:
Rename f

Reply:
@rohit-walkover please review the pr https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App/pull/4684 which contains review changes